rcnn/retrain_cnn.py

```diff
@@ -145,48 +148,43 @@ def create_image_lists(videos_dir, testing_percentage, validation_percentage):
     A dictionary containing an entry for each label subfolder, with images split
     into training, testing, and validation sets within each label.
   """
-  if not gfile.Exists(video_dir):
-    tf.logging.error("Video directory '" + video_dir + "' not found.")
-    return None
+
   result = {}
-  sub_dirs = [x[0] for x in gfile.Walk(video_dir)]
-  # The root directory comes first, so skip it.
-  is_root_dir = True
-  for sub_dir in sub_dirs:
-    if is_root_dir:
-      is_root_dir = False
-      continue
-    extensions = ['jpg', 'jpeg', 'JPG', 'JPEG']
-    file_list = []
-    dir_name = os.path.basename(sub_dir)
-    if dir_name == image_dir:
-      continue
-    tf.logging.info("Looking for images in '" + dir_name + "'")
-    for extension in extensions:
-      file_glob = os.path.join(image_dir, dir_name, '*.' + extension)
-      file_list.extend(gfile.Glob(file_glob))
-    if not file_list:
-      tf.logging.warning('No files found')
-      continue
-    if len(file_list) < 20:
+
+  for folder, labels in class_per_frame.items():
```

Comment:
Are we sure we have to use all class_per_frame items? The other scripts ask you which batches you want to use right?

Reply:
Since you are adding this on other scripts I'll wait till yours is merge to do it the same way.